client/elements/text/sc-simple-text.html

```diff
@@ -880,27 +880,22 @@ <h3>Could not find the sutta text.</h3>
           _markupComment(comment){
               comment = comment.replace('(', '').replace(')', '');
               const commentParts = comment.split(/,/);
-              let commentText = `<ul>`;
+              let commentText = '';
               for (let item in commentParts) {
                 const checkItem = this.editionsExpansionData[commentParts[item].replace(' ','')];
                 if (checkItem) {
                     commentText += `
-                                    <li>
                                       ${checkItem}
-                                    </li>
+                                    <br>
```

Comment:
I would put these on one line if it fits:

``commentText += `${checkItem}<br>`;``